shared/js/background/safari-wrapper.es6.js

```diff
@@ -1,3 +1,6 @@
+/* global safari:false */
```

Comment:
I ended up defining `safari` as a global in all the Safari-specific modules. Anywhere outside, I used `window.safari` instead.

My thinking was that it's guaranteed that Safari-specific modules will have `safari` defined as a global, but other files won't necessarily have that.